### PR TITLE
12.0 af29d67045486936f1ad5fe49b66e1c297f4560e

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: ir_attachment_s3/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.2.0`
+-------
+
+- **Improvement:** Save resized image to s3 instead of passing original (big) image
```
Comment (yelizariev) [inline ir_attachment_s3/doc/changelog.rst:4]: No need to test it
     "installable": False,

File: ir_attachment_url/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.1.7`
+-------
+
+- **Fix:** Product Variant were downloaded on server instead of passing url
```
Comment (yelizariev) [inline ir_attachment_url/doc/changelog.rst:4]: Has to be tested

File: web_preview/doc/changelog.rst [hunk 3]
```diff
@@ -1,4 +1,9 @@
+`1.0.1`
+-------
+
+**Fix:** Incorrect opening of the popup when using multiple binary fields in the model
```
Comment (yelizariev) [inline web_preview/doc/changelog.rst:4]: "installable": False,

File: web_website/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,7 @@
+`3.0.1`
+-------
+- **Fix:** Incorrect website priority after odoo updates https://github.com/odoo/odoo/commit/b6d32de31e0e18a506ae06dc27561d1d078f3ab1
```
Comment (yelizariev) [inline web_website/doc/changelog.rst:3]: tested in CI tests

File: ir_attachment_url/doc/changelog.rst (hunk 2 above)
Comment (yelizariev) [reply ir_attachment_url/doc/changelog.rst:4]: Doesn't work

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: ir_attachment_url/doc/changelog.rst (hunk 2 above)
Comment (yelizariev) [reply ir_attachment_url/doc/changelog.rst:4]: It's fixed in https://github.com/it-projects-llc/misc-addons/commit/84f4a442f9bff73a15337ac2b3afa5c288d124e0#diff-acb9c2027cd5315df02eb1eb6484f512, but I don't see those updates

@Rusllan

File: ir_attachment_url/models/ir_http.py [hunk 5]
```diff
@@ -20,40 +20,14 @@ class IrHttp(models.AbstractModel):
     _inherit = 'ir.http'
 
     @classmethod
-    def _find_field_attachment(cls, env, m, f, id):
```
Comment (yelizariev) [inline ir_attachment_url/models/ir_http.py]: Why do you remove it?

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by KolushovAlexandr** [comment]
Smth went wrong

**Pull request closed without merging**